keyboards/planck/keymaps/motform/keymap.c

```diff
@@ -0,0 +1,239 @@
+ ),
+
+/* Raise
+* ,-----------------------------------------------------------------------------------.
+* |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |  F7  |  F8  |  F9  |  F10 | F11  | F12  |
+* |------+------+------+------+------+-------------+------+------+------+------+------|
+* |   ^  |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  |  $   |
+* |------+------+------+------+------+------|------+------+------+------+------+------|
+* |      |      |      |   <  |   (  |      |      |  )   |   >  |Pg Up |Pg Dn |      |
+* |------+------+------+------+------+------+------+------+------+------+------+------|
+* |      |      |      |      |      |             |      | Next | Vol- | Vol+ | Play |
+* `-----------------------------------------------------------------------------------'
+*/
+[_RAISE] = LAYOUT_planck_grid
+(
+ KC_F1,   KC_F2,   KC_F3,   KC_F4,       KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,       KC_F10,  KC_F11,  KC_F12,
+ NO_CIRC, KC_1,    KC_2,    KC_3,        KC_4,    KC_5,    KC_6,    KC_7,    KC_8,        KC_9,    KC_0,    NO_DLR_MAC_V,
+ _______, KC_NO,   KC_NO,   NO_LESS_MAC, NO_LPRN, KC_NO,   KC_NO,   NO_RPRN, NO_GRTR_MAC, KC_PGUP, KC_PGDN, KC_NO,
+ _______, _______, _______, _______,     _______, _______, _______, _______, KC_MNXT,     KC_VOLD, KC_VOLU, KC_MPLY
+ ),
+
+/* Adjust (Lower + Raise)
+ * ,-----------------------------------------------------------------------------------.
+ * |      | Reset| Debug|      |      |      |      |      |      |      |      |      |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |      |      |Aud on|Audoff|AGnorm|AGswap|Colmak|      |      |      |      |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |Voice-|Voice+|Mus on|Musoff|MIDIon|MIDIof|TRMon |TRMoff|      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
+[_ADJUST] = LAYOUT_planck_grid
+(
+ _______, RESET,   DEBUG,   _______, _______, _______, _______, _______, _______,  _______, _______, _______,
+ _______, _______, MU_MOD,  AU_ON,   AU_OFF,  AG_NORM, AG_SWAP, COLEMAK, _______,  _______, _______, _______,
+ _______, MUV_DE,  MUV_IN,  MU_ON,   MU_OFF,  MI_ON,   MI_OFF,  TERM_ON, TERM_OFF, _______, _______, _______,
+ _______, _______, _______, _______, _______, _______, _______, _______, _______,  _______, _______, _______
+ )
+
+};
+
+uint32_t layer_state_set_user(uint32_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case COLEMAK:
```

Comment:
If you're only using the one base layer, then you don't actually need this keycode, then.